packages/kbn-interpreter/src/plugin/types/datatable.js

```diff
@@ -78,5 +78,32 @@ export const datatable = () => ({
         },
       };
     },
+    pointseries: datatable => {
+      // datatable columns are an array that looks like [{ name: "one", type: "string" }, { name: "two", type: "string" }]
+      // rows look like [{ one: 1, two: 2}, { one: 3, two: 4}, ...]
+      const validFields = ['x', 'y', 'color', 'size', 'text'];
```

Comment:
Could you do something like 

```javascript
Object.keys(pointseries().args)
```

 to make sure you don't have two places to edit these if something is added?

Reply:
Yeah, I was thinking about that too. I like that solution!